src/index.js

```diff
@@ -0,0 +1,30 @@
+import Bowser from 'bowser/bundled';
+import browserUtils from './browser-utils';
+import osUtils from './os-utils';
+import platformUtils from './platform-utils';
+
+const parser = Bowser.getParser(window.navigator.userAgent || '');
+
+export default {
+  // Browser flags
+  chrome: browserUtils.isChrome(parser),
+  chromium: browserUtils.isChromium(parser),
+  edge: browserUtils.isEdge(parser),
+  firefox: browserUtils.isFirefox(parser),
+  ie: browserUtils.isIE(parser),
+  opera: browserUtils.isOpera(parser),
+  safari: browserUtils.isSafari(parser),
+
+  // OS flags
+  android: osUtils.isAndroid(parser),
+  ios: osUtils.isIOS(parser),
+  iPadOS: osUtils.isIpadOS(parser),
+  linux: osUtils.isLinux(parser),
+  macOS: osUtils.isMacOS(parser) && !osUtils.isIpadOS(parser),
+  windows: osUtils.isWindows(parser),
+
+  // Platform flags
+  mobile: platformUtils.isMobile(parser),
+  tablet: platformUtils.isTablet(parser) || osUtils.isIpadOS(parser),
+  desktop: platformUtils.isDesktop(parser) && !osUtils.isIpadOS(parser),
```

Comment:
We bumped into a rendering issue with iOS 12.x vs iOS 13.x , so we need to know what exactly the version OS is.   Is it allowable to extend default export with such information? 
I.e.:

```js
...
osVersionName: get(parser, 'os.versionName', null), // it means lodash.get
...
```

Reply:
@aleksuk

Reply:
As an alternative, we could use `browser.satisfies` to export matches:
```js
...
iOS13Plus: browser.satisfies({
  macos: {
    safari: ">=13.0"
  },
}),
...
```

Reply:
It makes a sense back to discussing this issue later

Reply:
Would it be better to add `osVersion` key?